nakedret_test.go

```diff
@@ -21,25 +53,18 @@ func runNakedret(t *testing.T, filename string, maxLength uint, expected string)
 	if err := checkNakedReturns([]string{filename}, &maxLength, false); err != nil {
 		t.Fatal(err)
 	}
-	actual := string(logBuf.Bytes())
+	actual := logBuf.String()
+	fmt.Print(actual)
 	if expected != actual {
-		t.Errorf("Unexpected output:\n-----\n%s\n-----\n", actual)
+		// t.Errorf("Unexpected output:\n-----\ngot: \n%s\nexpected: \n%v\n-----\n", actual, expected)
```

Comment:
Feel free to swap over to this to include the "got"/"expected" wording - that might be more clear here.